### PR TITLE
SF 6 support

### PR DESCRIPTION
**Pull request by nguyenk**

### CODE REVIEW

File: Controller/GraphQLiteController.php [hunk 1]
```diff
@@ -78,6 +78,9 @@ public function handleRequest(Request $request): Response
             if (json_last_error() !== JSON_ERROR_NONE) {
                 throw new \RuntimeException('Invalid JSON received in POST body: '.json_last_error_msg());
             }
+            if (!is_array($parsedBody)){
+                throw new \RuntimeException('Expecting associative array from request, got ' . gettype($content));
```
Comment (andrew-demb) [inline Controller/GraphQLiteController.php:82]:
```suggestion
                throw new \RuntimeException('Expecting associative array from request, got ' . gettype($parsedBody));
```

**Comment by aszenz** [comment]
@nguyenk Any way i could help in moving this pr forward

**Comment by nguyenk** [comment]
Hello @aszenz, please check comment here https://github.com/thecodingmachine/graphqlite-bundle/issues/142#issuecomment-1476186593

I released the beta 6.0 but still no feedback. Did you test it? If it's working then I can merge.

**Comment by aszenz** [comment]
> I released the beta 6.0 but still no feedback. Did you test it? If it's working then I think we can merge it!

Thanks, that's great

I just upgraded the pkg in our app, we were already using symfony 6.2 using a forked pkg.

I had to change the minimum stability to dev for it to work
This is what composer did:
```
  - Upgrading thecodingmachine/graphqlite (v5.0.3 => v6.0.0)
  - Upgrading thecodingmachine/graphqlite-bundle (v5.4.0 => 6.0.x-dev e567930)
  - Upgrading thecodingmachine/graphqlite-symfony-validator-bridge (dev-jerowork/upgrade-symfony-to-6 fb5515e => 6.0.x-dev 7b1ee6e)
```

Our ci pipeline succeeded and I tested few screens manually which use graphql, everything seems to work fine.

I guess we need to release a new version for both `graphqlite-bundle` and `graphqlite-symfony-validator-bridge` pkgs

I do see some deprecations on symfony 6 though for this pkg, I guess we could fix them after release as well.

- Method "Overblog\GraphiQLBundle\Config\GraphiQLControllerEndpoint::getBySchema()" might add "string" as a native return type declaration in the future. Do the same in implementation "TheCodingMachine\GraphQLite\Bundle\GraphiQL\EndpointResolver" now to avoid errors or add an explicit @return annotation to suppress this message.
- Method "Overblog\GraphiQLBundle\Config\GraphiQLControllerEndpoint::getDefault()" might add "string" as a native return type declaration in the future. Do the same in implementation "TheCodingMachine\GraphQLite\Bundle\GraphiQL\EndpointResolver" now to avoid errors or add an explicit @return annotation to suppress this message.
- Since symfony/console 6.1: Relying on the static property "$defaultName" for setting a command name is deprecated. Add the "Symfony\Component\Console\Attribute\AsCommand" attribute to the "TheCodingMachine\GraphQLite\Bundle\Command\DumpSchemaCommand" class instead.
- User Deprecated: Method "ArrayAccess::offsetGet()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "GraphQL\Language\AST\NodeList" now to avoid errors or add an explicit @return annotation to suppress this message.
- Deprecated: Using ${var} in strings is deprecated, use {$var} instead

**Comment by nguyenk** [comment]
@aszenz merged to master, thank you again for your help

**Pull request merged**